client.go

```diff
@@ -1197,7 +1197,7 @@ func (c *Client) GetAlias(alias string) string {
 
 // Init creates a container from either a fingerprint or an alias; you must
 // provide at least one.
-func (c *Client) Init(name string, imgremote string, image string, profiles *[]string, config map[string]string, ephem bool) (*Response, error) {
+func (c *Client) Init(name string, imgremote string, image string, profiles *[]string, config map[string]string, devices shared.Devices, ephem bool) (*Response, error) {
```

Comment:
FWIW, at a certain point it makes sense to use an args struct instead of adding more arguments to the signature.  Usually that point is when you have more than 2 arguments.  The benefits of an args struct include a cleaner signature, backward-compatibility when adding more args, and what amounts to keyword argument support for callers.

If Client.Init() is something you expect folks to use (which I presume is the case) then it would be better to switch to an args struct now and get it over with. :)